tests/openwrt/test_modemmanager_dsa.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from netjsonconfig import OpenWrt
+from netjsonconfig.utils import _TabsMixin
+
+
+class TestModemManager(unittest.TestCase, _TabsMixin):
+    maxDiff = None
+
+    _modemmanager_interface_netjson = {
+        "interfaces": [
+            {
+                "name": "wwan0",
+                "type": "modem-manager",
+                "apn": "apn.vodafone.com",
+                "pin": "1234",
+                "device": "/sys/devices/platform/ahb/1b000000.usb/usb1/1-1",
+                "username": "user123",
+                "password": "pwd123456",
+                "metric": 50,
+                "iptype": "ipv4v6",
+                "lowpower": False,
+                "mtu": 1500,
```

Comment:
let's add the signal rate here and to the schema please, to address #226 since these issues are related.